gocli/cmd/run.go

```diff
@@ -13,14 +20,10 @@ import (
 	"github.com/spf13/pflag"
 	"golang.org/x/net/context"
 	"kubevirt.io/kubevirtci/gocli/docker"
-	"os"
-	"os/signal"
-	"path/filepath"
-	"strconv"
-	"sync"
 )
 
 const NFSGaneshaImage = "docker.io/janeczku/nfs-ganesha@sha256:17fe1813fd20d9fdfa497a26c8a2e39dd49748cd39dbb0559df7627d9bcf4c53"
+const CephImage = "docker.io/ceph/daemon:latest-luminous"
```

Comment:
Pleasue use a non-floating tag. This one looks like it can float. If there is no non-floating tag, you can use the shasum.

Reply:
done